app/models/application.rb

```diff
@@ -149,12 +149,12 @@ def full_name
     [title, first_name, last_name].join(' ')
   end
 
-  def known_over_61?
+  def known_partner_over_61?
```

Comment:
I think this should be applicant_known_over_61? if it has to be renamed.

The code checks on the applicants date of birth, and stops us hitting the partner_over_61 section.